_includes/footer.html

```diff
@@ -3,7 +3,7 @@
     {% include logos.html %}
     <br /><br />
     <p>
-      {{ site.title }} is organized with ❤. Find us on   
+      {{ site.title }} is produced by <a href="https://compiler.la/">Compiler</a>>. Find us on   
```

Comment:
Tiny little typo, but otherwise this looks great!

```suggestion
      {{ site.title }} is produced by <a href="https://compiler.la/">Compiler</a>. Find us on   
```

Reply:
Updated. I also removed the twitter too.